readme.md

```diff
@@ -1,36 +1,154 @@
-# Local and social authentication with Laravel and Nuxt 
```

Comment:
Thanks a lot for this, looks really good and it will hopefully save people a lot of time 👍

Reply:
@fritsvt my pleasure 👍 👍